docs/howto/mock-services.md

```diff
@@ -474,7 +474,7 @@ You can find more examples in the `examples` and `test-tests` subproject:
 - [ComposedEmptyMockSpec][link-gh-composed-empty-mock-spec]
 - [PolyMockSpec][link-gh-poly-mock-spec]
 
-[doc-use-modules-and-layers]: use-modules-and-layers.md
+[doc-use-modules-and-layers]: ../datatypes/contextual/index.md
```

Comment:
We also need to change the title of the page we are referring to that. The following sentence doesn't reflect that:

> We'll be assuming you've read How to use modules and layers guide.

Reply:
That makes sense. Does this sound better?
"We'll be assuming you read about modules and layers in the contextual types guide."

Reply:
Yes, it sounds better.

Reply:
Done.